FLEDGE.md

```diff
@@ -223,8 +223,16 @@ const myAuctionConfig = {
     ...
   ],
   'signal': /* optionally, an AbortSignal */...,
+  resolveToConfig: /* optionally, a boolean */...,
 };
-const auctionResultPromise = navigator.runAdAuction(myAuctionConfig);
+const auctionResult = await navigator.runAdAuction(myAuctionConfig);
+
+// If `result` is a `FencedFrameConfig` object, it must be used with fenced frame
```

Comment:
We cannot have `FencedFrameConfig` link to anything here, because this is in a code block. But I'll make the `with a fenced frame` suggestion you mentioned.